README.md

```diff
@@ -43,69 +43,95 @@ remotes::install_github("mrc-ide/frogger", upgrade = FALSE)
+| incrr_age          | incidence.age_rate_ratio                 | HIV incidence rate ratio by age (for each sex)                                                                                                                        |
+| cd4_initdist       | natural_history.cd4_initial_distribution | Distribution of infections by CD4 category upon infection                                                                                                             |
+| cd4_prog           | natural_history.cd4_progression          | Probability of progressing from 1 CD4 stage to the next by age and sex                                                                                                |
+| cd4_mort           | natural_history.cd4_mortality            | Probability of mortality by CD4 stage, age and sex                                                                                                                    |
+| scale_cd4_mort     | natural_history.scale_cd4_mortality      | If 1 then scale HIV related mortality (i.e. cd4_mortality) as a proportion of number of people with HIV and over the number with HIV and on ART at this disease stage |
+| art_mort           | art.mortality                            | Probability of mortality by treatment stage, CD4 stage, age and sex                                                                                                   |
+| artmx_timerr       | art.mortaility_rate_ratio                | HIV-related mortality rate ratios by treatment stage                                                                                                                  |
+| art15plus_num      | art.adults_on_art                        | Time series of # or % of adult PLHIV on ART by sex                                                                                                                    |
+| art15plus_isperc   | art.adults_on_art_is_percent             | Time series, TRUE if art15plus_num is a %, FALSE if it is a #                                                                                                         |
+| artcd4elig_idx     | art.idx_cd4_elig                         | The index of the CD4 count at which people are eligible for ART by time step                                                                                          |
+| art_alloc_mxweight | art.initiation_mortality_weight          | Weighting for extent that expected mortality guides ART uptake                                                                                                        |
+| art_dropout        | art.dropout                              | ART dropout rate at each time step                                                                                                                                    |
```

Comment:
Suggest maybe `art_dropout`.  Reason being that an instance of the class might be assigned to whatever name; so it might not always be read as `art.dropout` (with the class name) in the code, so I wonder if variable names should be 'complete'.  (Curious feedback from you and Rich about best practice on this)

In the 'Description' column, suggest "Annual HIV dropout rate" ("per time step" ambiguous whether it is per 0.1 year time step or annualized)

Reply:
I think using the `Art` struct to namespace this is one of the big advantages of splitting the parameters up. the class will only be instantiated once in the `Parameters` struct so will always be called `art`. There is an example of usage here https://github.com/mrc-ide/frogger/blob/main/inst/include/model_simulation.hpp#L170